examples/matRad_example15_paretoOptimization.m

```diff
@@ -0,0 +1,118 @@
+%% Example: Photon Treatment Plan
```

Comment:
The result wont make sense, since the optimization terminates after 10(?) iterations and we need Pareto optimality in each plan which is only achieved when running for longer times.